Modules/mmapmodule.c

```diff
@@ -656,6 +657,7 @@ is_resizeable(mmap_object *self)
     return 0;
 
 }
+#endif /* defined(MS_WINDOWS) || defined(HAVE_MREMAP) */
```

Comment:
We usually write this simpler:
```suggestion
#endif /* MS_WINDOWS || HAVE_MREMAP */
```